modules/cpms-changing-openstack-flavor-type.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+// * machine_management/control_plane_machine_management/cpmso-using.adoc
+
+:_content-type: PROCEDURE
+[id="cpms-changing-openstack-flavor-type_{context}"]
+= Changing the {rh-openstack} compute flavor by using a control plane machine set
```

Comment:
Are we using the flavor as an example?

Reply:
Not sure what you mean. Could you rephrase?

Reply:
I mean we can use other CPMS attributes and the flavor is just an example , right?

Reply:
@itzikb-redhat It's what the previous writer included in their work--not 100% on it myself. I'm not sure how important the flavor task is specifically, nor if there are limitations on which attributes you can change. @EmilienM ?

Reply:
the flavor example is because CPMS helps with vertical scaling too. You can easily scale your control plane vertically by changing the flavors and obtain better perfs.

Reply:
Can we state with something like that - The scaling up is possible with CPMS by changing the flavor?

Reply:
https://github.com/openshift/openshift-docs/pull/66212/commits/67aff27738a7b6429f828d88be92f452c9573d99